demo/README.md

```diff
@@ -25,6 +25,8 @@ w2 as (PARTITION BY passenger_count ORDER BY pickup_datetime ROWS_RANGE BETWEEN
 
 ## 2. Demo with The Cluster Mode
 
+> :warning: Required docker version >= 18.03
```

Comment:
it's docker desktop for macOS intel ?
not docker package for linux

Reply:
it's the docker-engine/moby's version.

Reply:
docker package you mean docker client? or docker-ce?

Reply:
whichever contains docker engine. that's fine